packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -134,8 +115,8 @@ export function RobotInfo({ robot }: RobotInfoProps): JSX.Element {
           <Button
             disableElevation
             variant="outlined"
-            classes={{ root: classes.root, disabled: classes.disabled }}
-            disabled
+            className={classes.button}
+            disableRipple={true}
```

Comment:
Why disable the ripple effect?

Reply:
So as to mimic a disabled button without having to modify the classes to change colors to make it more visible.

Reply:
In that case, you should set the `component` prop to `div` so as to correctly let the accessibility engine know that it is not a button.